frontend/amundsen_application/static/js/components/ResourceListItem/TableListItem/index.tsx

```diff
@@ -31,6 +32,9 @@ const TableListItem: React.FC<TableListItemProps> = ({ table, logging }) => (
     <Link
       className="resource-list-item table-list-item"
       to={getLink(table, logging)}
+      onClick={(e) =>
+        logClick(e, { target_id: 'table_list_item', value: logging.source })
+      }
```

Comment:
Every logClick we introduce is more technical debt we are accruing on, let's try to avoid it.